tone/tone.d.ts

```diff
@@ -0,0 +1,1104 @@
+// Type definitions for TONE.JS
+// Project: https://github.com/TONEnoTONE/Tone.js
+// Definitions by: Luke Phillips <https://github.com/lukephills>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="web-audio.d.ts" />
```

Comment:
https://github.com/borisyankov/DefinitelyTyped/blob/master/webaudioapi/waa.d.ts
please check this definition file. same as it?